src/ingest-pipeline/airflow/dags/scan_and_begin_processing.py

```diff
@@ -199,8 +209,10 @@ def flex_maybe_spawn(**kwargs):
         ctx = kwargs['dag_run'].conf
         pprint(ctx)
         run_validation_retcode = int(kwargs['ti'].xcom_pull(task_ids="run_validation"))
-        md_extract_retcode = int(kwargs['ti'].xcom_pull(task_ids="run_md_extract"))
-        md_consistency_retcode = int(kwargs['ti'].xcom_pull(task_ids="md_consistency_tests"))
+        md_extract_retcode = kwargs['ti'].xcom_pull(task_ids="run_md_extract")
```

Comment:
I'm curious why you broke this into two statements instead of adding the `or '0'` to lines 202-203 within the `int()` call.